crates/experimental-api/src/rpc.rs

```diff
@@ -171,9 +173,27 @@ impl gen::Rpc for Context {
     async fn call(
         &self,
         request: FunctionCall,
-        block_id: BlockId,
+        _block_id: BlockId,
```

Comment:
nit: Do we still need the `block_id` parameter if we don't use it?

Reply:
We don't in this particular case, but it is a part of the RPC API